### PR TITLE
Listed my project rockpaperscissor.yml

### PR DESCRIPTION
**Pull request by zetabug**

### CODE REVIEW

File: _data/projects/rockpaperscissor.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  name: hacktoberfest
+  link: https://github.com/zetabug/rock-paper-scissors-game/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest
+goodfirstissue:
+  name : good first issue 
+  link : https://github.com/zetabug/rock-paper-scissors-game/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
+helpwanted :
+  name: help wanted 
+  link : https://github.com/zetabug/rock-paper-scissors-game/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
+upforgrabs :
+  name: up for grabs
```
Comment (shiftkey) [inline _data/projects/rockpaperscissor.yml]: 🤔 this section is the only one that will be used on the site, but you can set `name` and `link` to be whichever label you prefer. Which label did you want to use?

**Comment by zetabug** [comment]
I resolved the problems that were there, please review and merge this PR fast. @shiftbot @shiftkey

File: _data/projects/rockpaperscissor.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Rock, Paper & Scissor
+desc: A simple fully responsive rock,paper and scissors game developed using HTML,CSS & JAVASCRIPT.
+site: https://rock-paper-scissors-game0302.netlify.app/
+tags:
+  - hacktoberfest
+  - javascript
+  - up-for-grabs
+  - front-end-development
+  - open-source
+hacktoberfest:
```
Comment (shiftkey) [inline _data/projects/rockpaperscissor.yml]: :art: this needs to be updated to address that CI warning

```suggestion
upforgrabs:
```

File: _data/projects/rockpaperscissor.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  name: hacktoberfest
+  link: https://github.com/zetabug/rock-paper-scissors-game/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest
```
Comment (shiftkey) [inline _data/projects/rockpaperscissor.yml]: :art: a shorter link

```suggestion
  link: https://github.com/zetabug/rock-paper-scissors-game/labels/hacktoberfest
```

**Pull request merged**